docs/developer/watcher-local-test.md

```diff
@@ -0,0 +1,136 @@
+# Test watcher locally in two-cluster-setup using K3D
+
+### Contents
+
+### Export customizable attributes
+```shell
+export KLM_PATH=<local-path-of-kyma-lifecycle-manager-project> && \
+export KLM_VERSION=<docker-image-tag-for-local-testing>  && \
+export KUBE_CONFIG_DIR=<path-to-export-local-kubeconfigs-to>
+```
+
+### Create KCP cluster
+Run the following command to create a local control-plane (KCP) cluster:
+```shell
+k3d cluster create kcp-local --port 9080:80@loadbalancer \
+--registry-create k3d-registry.localhost:0.0.0.0:5111 \
```

Comment:
```suggestion
--registry-create k3d-kyma-registry.localhost:0.0.0.0:5111 \
```
since this is the same registry name as with Kyma CLI. On that note, maybe we could/should switch to that?